moveit_experimental/collision_distance_field_ros/CMakeLists.txt

```diff
@@ -12,9 +12,9 @@ include($ENV{ROS_ROOT}/core/rosbuild/rosbuild.cmake)
 rosbuild_init()
 
 #set the default path for built executables to the "bin" directory
-set(EXECUTABLE_OUTPUT_PATH ${PROJECT_SOURCE_DIR}/bin)
+set(EXECUTABLE_OUTPUT_PATH ${CATKIN_GLOBAL_BIN_DESTINATION})
 #set the default path for built libraries to the "lib" directory
-set(LIBRARY_OUTPUT_PATH ${PROJECT_SOURCE_DIR}/lib)
+set(LIBRARY_OUTPUT_PATH ${CATKIN_GLOBAL_LIB_DESTINATION})
```

Comment:
This file is still rosbuild and does not build anything either way.
It doesn't make sense to use catkin variables here.

I would propose to comment out the last 4 functional lines in this file if this really triggers catkin_lint warnings.